testutil/test_helpers.go

```diff
@@ -317,6 +364,10 @@ func NewProviderConsumerCoordinator(t *testing.T) *ibctesting.Coordinator {
 	coordinator.Chains[chainID] = ibctesting.NewTestChainWithValSet(t, coordinator,
 		chainID, providerChain.Vals, providerChain.Signers)
 
+	chainID = ibctesting.GetChainID(4)
```

Comment:
please move all magic numbers to constants

Reply:
It is not a magic constant, but the common way we init new chains using testutil. i've switched all of such entries to constants but not quite sure if  it has improved readability